scripts/main.py

```diff
@@ -480,6 +479,10 @@ def main():
             # for training.
             args.noisybw = gather_files_from_cmdline(args.noisybw,
                                                      extension=".bw")
+            if args.mode == "eval":
+                args.cleanbw = gather_files_from_cmdline(args.cleanbw,
```

Comment:
there's already a condition for `mode == eval` a few lines above - can this be placed there?

Reply:
good catch, changed it.